### ISSUE TITLE
Weather dashboard page Blueprint

### ISSUE THREAD

**Issue by FabryStyle** [dwainscheeren/dwains-dashboard-blueprints#110]
Hi
It's about 3 days that I'm trying to understand why this blueprint page doesn't work and I'm feel really DUMB right now.

I follow and triple check all the instructions multiple times but I'm still getting the same error all the time, please:

![image](https://user-images.githubusercontent.com/121764722/210896554-02756cc3-d348-40b7-81d3-3d39bbd31fcf.png)

**Comment by LordNex** [comment]
Yep most of them are dead. If your lucky sometimes you can actually get it to display something if you leave all the OWM entities with their default name. See issue #104

**Comment by MichelJourdain** [comment]
The problem was that adding Blueprint no longer works as expected.  Normally when adding a Blueprint that contains a Template, it is added automatically.  For now you can add a file in the /**config/dwains-dashboard/button_card_templates/blueprints/** directory.  It should contain the following code.  

,,,

weather_dashboard_blueprint_heading:
  show_icon: false
  tyles:
      card:
        - box-shadow: none
        - padding: 6px 0 0 0
        - background: transparent
        - width: 90px
      name:
        - font-weight: bold
        - justify-self: start
        - font-size: 16px
        - padding-left: 1px
,,,

Restart Home assistant.

![0F0F1507-AFEE-4C0F-B483-8850C36ADE17](https://user-images.githubusercontent.com/83040228/211048873-fdea77f7-00d2-4ed3-b91d-31823d1994bf.jpeg)

**Comment by vinnie1234** [comment]
Add a pull request that fix all the problems and changed the documentation!

**Comment by vinnie1234** [comment]
Fixes are merged with #119. Are there still problems left?

**Comment by LordNex** [comment]
> Fixes are merged with #119. Are there still problems left? 

Still the same for me. I haven't seen an update or anything so I can't see anything has changed

**Comment by vinnie1234** [comment]
Dit you try adding the blueprint again? 

Je can also go do:
Config/dwains-dashboard/blueprints/weather_dashboard_page_blueprint.yaml and copie the blueprint in that file. And the card in Config/dwains-dashboard/button_card_templates/blueprints/weather_dashboard_page_blueprint.yaml

Then edit youre page. When you submit the style has to be fixed.

**Comment by LordNex** [comment]
I've tried that. I still don't get an area to paste the blueprint in for one. Even if I manually create the blueprint in the directory, reload DD, and then try and add the page, I get a page much like the screenshot above that won't let me select the proper inputs and if saved says the blueprint can't be found. 

I've deleted DD completely from my system and redownloaded it from HACS and it's the same. So unless he's pushed out a newer version that I don't have the issue is still there

**Comment by vinnie1234** [comment]
Do you use v1.2 for the blueprint and add all the config settings?

When you are member of the discord, ask the question there and add @vinnie1234. Then I can help you in the weekend. (When you speak dutch, it can in the Dutch part)

**Comment by nwithan8** [comment]
If you want, I just took the blueprint, ripped out all the customization capabilities from it and hard-coded it to sensors (if you follow the setup instructions for that blueprint and don't manually change entity IDs, this should work out of the box).

Add this as a normal Lovelace Card rather than as a blueprint
https://github.com/nwithan8/configs/blob/main/home_assistant/panels/weather.yaml

Thanks to @rwilson131 for making this in the first place!

**Comment by rwilson131** [comment]
@nwithan8 I have added the mushroom chip cards,  much better.  Thanks.  
@MichelJourdain do I need to do something different on my end for the installation?  

@LordNex @FabryStyle it no longer uses button card with the changes from @nwithan8 I do not believe that fixes the issues @MichelJourdain is referring to with DD and new blueprints.  

I have left all of the customization and updated some minor typos.  I have created a pull request, so hopefully it will be available shortly.  

https://github.com/rwilson131/dwains-dashboard-blueprints/tree/main/page-blueprints/weather-dashboard-openweathermap
